### DISCUSSION TITLE
Bash Error: unknown command "shell" for "oh-my-posh" - Linux Lite

### DISCUSSION THREAD

**Discussion by MiguelAngelGarcerantAmador** [JanDeDobbeleer/oh-my-posh#2914] [Troubleshoot]
@### ** Bash Error: unknown command "shell" for "oh-my-posh"** 

**Sistema Operativo que utilizó?**
Linux - Distro(Linux Lite)

**Shell que utilizó?**
Bash

Instale Oh-My-Posh de manera manual según la documentación  de la pagina https://ohmyposh.dev/docs/installation/linux y dando como resultado esta imagen:

![Error shell ](https://user-images.githubusercontent.com/107361498/194729320-24faa86b-bd5b-4719-84ff-d4711f8e2ab2.png)

Esta fue la configuración que añadí al .bashrc
> **eval "$(oh-my-posh init bash --config ~/.poshthemes/jandedobbeleer.omp.json)"**

como se puede ver ya esta funcionando pero cada vez que escribo un comando en la terminal aparece el mensaje de **Error: unknown command "shell" for "oh-my-posh"** lo cual es muy incomodo.

Seguido de esto nuevamente aparece el error **Error: unknown command "shell" for "oh-my-posh"**, quisiera saber si hice algún procedimiento anormal en la instalación, solicito su ayuda ya que no he podido encontrar el error.

Anexo el debug de oh-my-posh hecho con el Comando - `oh-my-posh debug`

Versión: 12.0.1

Segments:

ConsoleTitle(true)   -   1 ms - bash in garcerant
session(true)        -   2 ms -  garcerant 
path(true)           -   0 ms -    ~ 
git(false)           -   0 ms - 
node(false)          -   0 ms - 
go(false)            -   0 ms - 
julia(false)         -   0 ms - 
python(false)        -   0 ms - 
ruby(false)          -   0 ms - 
azfunc(false)        -   0 ms - 
aws(false)           -   0 ms - 
root(false)          -   0 ms - 
executiontime(true)  -   0 ms -  ﮫ0ms⠀
exit(true)           -   0 ms -   
shell(true)          -   0 ms -   bash 
ytm(false)           -   0 ms - 
battery(false)       -   0 ms - 
time(true)           -   0 ms -  17:43:17 

Run duration: 20.647922ms

Cache path: /home/garcerant/.cache/oh-my-posh

Config path: /home/garcerant/.poshthemes/jandedobbeleer.omp.json

Logs:

2022/10/08 17:43:17 debug: Getenv

2022/10/08 17:43:17 Getenv duration: 18.081µs, args: XDG_CACHE_HOME
2022/10/08 17:43:17 CachePath duration: 52.577µs, args: 
2022/10/08 17:43:17 debug: Getenv
/home/garcerant/.poshthemes/jandedobbeleer.omp.json
2022/10/08 17:43:17 Getenv duration: 45.498µs, args: POSH_THEME
2022/10/08 17:43:17 resolveConfigPath duration: 60.847µs, args: 
2022/10/08 17:43:17 Init duration: 9.319196ms, args: 
2022/10/08 17:43:17 Flags duration: 271ns, args: 
2022/10/08 17:43:17 config.loadConfig duration: 2.226352ms, args: 
2022/10/08 17:43:17 Flags duration: 257ns, args: 
2022/10/08 17:43:17 debug: Getenv

2022/10/08 17:43:17 Getenv duration: 3.336µs, args: OMP_CACHE_DISABLED
2022/10/08 17:43:17 Root duration: 1.609µs, args: 
2022/10/08 17:43:17 debug: Shell
process name: bash
2022/10/08 17:43:17 Shell duration: 1.083767ms, args: 
2022/10/08 17:43:17 ErrorCode duration: 121ns, args: 
2022/10/08 17:43:17 debug: Getenv

2022/10/08 17:43:17 Getenv duration: 2.316µs, args: WSL_DISTRO_NAME
2022/10/08 17:43:17 IsWsl duration: 5.341µs, args: 
2022/10/08 17:43:17 GOOS duration: 180ns, args: 
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 25.674µs, args: 
2022/10/08 17:43:17 GOOS duration: 141ns, args: 
2022/10/08 17:43:17 GOOS duration: 55ns, args: 
2022/10/08 17:43:17 GOOS duration: 49ns, args: 
2022/10/08 17:43:17 GOOS duration: 48ns, args: 
2022/10/08 17:43:17 GOOS duration: 48ns, args: 
2022/10/08 17:43:17 GOOS duration: 48ns, args: 
2022/10/08 17:43:17 GOOS duration: 49ns, args: 
2022/10/08 17:43:17 GOOS duration: 48ns, args: 
2022/10/08 17:43:17 GOOS duration: 48ns, args: 
2022/10/08 17:43:17 GOOS duration: 48ns, args: 
2022/10/08 17:43:17 GOOS duration: 49ns, args: 
2022/10/08 17:43:17 debug: User
garcerant
2022/10/08 17:43:17 User duration: 2.857µs, args: 
2022/10/08 17:43:17 debug: Host
Garcerant
2022/10/08 17:43:17 Host duration: 6.399µs, args: 
2022/10/08 17:43:17 GOOS duration: 72ns, args: 
2022/10/08 17:43:17 TemplateCache duration: 1.231124ms, args: 
2022/10/08 17:43:17 Flags duration: 209ns, args: 
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 7.96µs, args: 
2022/10/08 17:43:17 GOOS duration: 113ns, args: 
2022/10/08 17:43:17 debug: Getenv

2022/10/08 17:43:17 Getenv duration: 2.82µs, args: SSH_CONNECTION
2022/10/08 17:43:17 debug: Getenv

2022/10/08 17:43:17 Getenv duration: 2.067µs, args: SSH_CLIENT
2022/10/08 17:43:17 TemplateCache duration: 123ns, args: 
2022/10/08 17:43:17 TemplateCache duration: 225ns, args: 
2022/10/08 17:43:17 Shell duration: 317ns, args: 
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 3.543µs, args: 
2022/10/08 17:43:17 GOOS duration: 88ns, args: 
2022/10/08 17:43:17 Shell duration: 99ns, args: 
2022/10/08 17:43:17 Shell duration: 50ns, args: 
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 1.299µs, args: 
2022/10/08 17:43:17 GOOS duration: 70ns, args: 
2022/10/08 17:43:17 PathSeparator duration: 97ns, args: 
2022/10/08 17:43:17 GOOS duration: 56ns, args: 
2022/10/08 17:43:17 GOOS duration: 98ns, args: 
2022/10/08 17:43:17 GOOS duration: 55ns, args: 
2022/10/08 17:43:17 PathSeparator duration: 48ns, args: 
2022/10/08 17:43:17 GOOS duration: 41ns, args: 
2022/10/08 17:43:17 PathSeparator duration: 51ns, args: 
2022/10/08 17:43:17 GOOS duration: 40ns, args: 
2022/10/08 17:43:17 PathSeparator duration: 58ns, args: 
2022/10/08 17:43:17 GOOS duration: 42ns, args: 
2022/10/08 17:43:17 PathSeparator duration: 76ns, args: 
2022/10/08 17:43:17 GOOS duration: 61ns, args: 
2022/10/08 17:43:17 debug: Getenv

2022/10/08 17:43:17 Getenv duration: 1.943µs, args: WSL_DISTRO_NAME
2022/10/08 17:43:17 IsWsl duration: 5.077µs, args: 
2022/10/08 17:43:17 StackCount duration: 101ns, args: 
2022/10/08 17:43:17 DirIsWritable duration: 14.358µs, args: 
2022/10/08 17:43:17 TemplateCache duration: 116ns, args: 
2022/10/08 17:43:17 TemplateCache duration: 194ns, args: 
2022/10/08 17:43:17 Shell duration: 149ns, args: 
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 5.284µs, args: 
2022/10/08 17:43:17 GOOS duration: 95ns, args: 
2022/10/08 17:43:17 debug: Getenv

2022/10/08 17:43:17 Getenv duration: 1.84µs, args: WSL_DISTRO_NAME
2022/10/08 17:43:17 IsWsl duration: 4.763µs, args: 
2022/10/08 17:43:17 GOOS duration: 55ns, args: 
2022/10/08 17:43:17 debug: CommandPath
/usr/bin/git
2022/10/08 17:43:17 CommandPath duration: 50.758µs, args: git
2022/10/08 17:43:17 HasCommand duration: 54.733µs, args: git
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 1.445µs, args: 
2022/10/08 17:43:17 error: HasParentFilePath
stat /.git: no such file or directory
2022/10/08 17:43:17 HasParentFilePath duration: 27.947µs, args: .git
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 3.399µs, args: 
2022/10/08 17:43:17 GOOS duration: 84ns, args: 
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 1.386µs, args: 
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 3.303µs, args: 
2022/10/08 17:43:17 GOOS duration: 85ns, args: 
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 1.48µs, args: 
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 16.518µs, args: 
2022/10/08 17:43:17 GOOS duration: 76ns, args: 
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 2.074µs, args: 
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 3.253µs, args: 
2022/10/08 17:43:17 GOOS duration: 83ns, args: 
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 1.394µs, args: 
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 1.324µs, args: 
2022/10/08 17:43:17 PathSeparator duration: 66ns, args: 
2022/10/08 17:43:17 debug: HasFiles
false
2022/10/08 17:43:17 HasFiles duration: 151.363µs, args: *.py
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 1.504µs, args: 
2022/10/08 17:43:17 PathSeparator duration: 95ns, args: 
2022/10/08 17:43:17 debug: HasFiles
false
2022/10/08 17:43:17 HasFiles duration: 128.473µs, args: *.ipynb
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 1.501µs, args: 
2022/10/08 17:43:17 PathSeparator duration: 81ns, args: 
2022/10/08 17:43:17 debug: HasFiles
false
2022/10/08 17:43:17 HasFiles duration: 18.05µs, args: pyproject.toml
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 1.466µs, args: 
2022/10/08 17:43:17 PathSeparator duration: 72ns, args: 
2022/10/08 17:43:17 debug: HasFiles
false
2022/10/08 17:43:17 HasFiles duration: 13.279µs, args: venv.bak
2022/10/08 17:43:17 debug: HasFolder
false
2022/10/08 17:43:17 HasFolder duration: 21.737µs, args: .venv
2022/10/08 17:43:17 debug: HasFolder
false
2022/10/08 17:43:17 HasFolder duration: 7.429µs, args: venv
2022/10/08 17:43:17 debug: HasFolder
false
2022/10/08 17:43:17 HasFolder duration: 7.179µs, args: virtualenv
2022/10/08 17:43:17 debug: HasFolder
false
2022/10/08 17:43:17 HasFolder duration: 7.001µs, args: env
2022/10/08 17:43:17 debug: HasFolder
false
2022/10/08 17:43:17 HasFolder duration: 6.923µs, args: venv-win
2022/10/08 17:43:17 debug: HasFolder
false
2022/10/08 17:43:17 HasFolder duration: 7.383µs, args: pyenv-win
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 3.319µs, args: 
2022/10/08 17:43:17 GOOS duration: 110ns, args: 
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 1.382µs, args: 
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 3.322µs, args: 
2022/10/08 17:43:17 GOOS duration: 103ns, args: 
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 1.336µs, args: 
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 3.046µs, args: 
2022/10/08 17:43:17 GOOS duration: 89ns, args: 
2022/10/08 17:43:17 debug: Getenv

2022/10/08 17:43:17 Getenv duration: 1.917µs, args: AWS_VAULT
2022/10/08 17:43:17 debug: Getenv

2022/10/08 17:43:17 Getenv duration: 1.572µs, args: AWS_PROFILE
2022/10/08 17:43:17 debug: Getenv

2022/10/08 17:43:17 Getenv duration: 2.26µs, args: AWS_REGION
2022/10/08 17:43:17 debug: Getenv

2022/10/08 17:43:17 Getenv duration: 3.31µs, args: AWS_DEFAULT_REGION
2022/10/08 17:43:17 debug: Getenv

2022/10/08 17:43:17 Getenv duration: 1.591µs, args: AWS_CONFIG_FILE
2022/10/08 17:43:17 error: FileContent
open /home/garcerant/.aws/config: no such file or directory
2022/10/08 17:43:17 FileContent duration: 11.075µs, args: /home/garcerant/.aws/config
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 3.04µs, args: 
2022/10/08 17:43:17 GOOS duration: 95ns, args: 
2022/10/08 17:43:17 Root duration: 1.187µs, args: 
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 3.069µs, args: 
2022/10/08 17:43:17 GOOS duration: 85ns, args: 
2022/10/08 17:43:17 ExecutionTime duration: 160ns, args: 
2022/10/08 17:43:17 TemplateCache duration: 115ns, args: 
2022/10/08 17:43:17 TemplateCache duration: 205ns, args: 
2022/10/08 17:43:17 Shell duration: 138ns, args: 
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 3.215µs, args: 
2022/10/08 17:43:17 GOOS duration: 95ns, args: 
2022/10/08 17:43:17 ErrorCode duration: 101ns, args: 
2022/10/08 17:43:17 TemplateCache duration: 104ns, args: 
2022/10/08 17:43:17 TemplateCache duration: 172ns, args: 
2022/10/08 17:43:17 Shell duration: 104ns, args: 
2022/10/08 17:43:17 TemplateCache duration: 204ns, args: 
2022/10/08 17:43:17 Flags duration: 174ns, args: 
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 3.274µs, args: 
2022/10/08 17:43:17 GOOS duration: 90ns, args: 
2022/10/08 17:43:17 Shell duration: 109ns, args: 
2022/10/08 17:43:17 Flags duration: 83ns, args: 
2022/10/08 17:43:17 TemplateCache duration: 98ns, args: 
2022/10/08 17:43:17 TemplateCache duration: 190ns, args: 
2022/10/08 17:43:17 Shell duration: 136ns, args: 
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 22.763µs, args: 
2022/10/08 17:43:17 GOOS duration: 102ns, args: 
2022/10/08 17:43:17 debug: HTTPRequest
GET /query HTTP/1.1
Host: 127.0.0.1:9863
User-Agent: Go-http-client/1.1
Accept-Encoding: gzip

2022/10/08 17:43:17 error: HTTPRequest
Get "http://127.0.0.1:9863/query": dial tcp 127.0.0.1:9863: connect: connection refused
2022/10/08 17:43:17 HTTPRequest duration: 558.902µs, args: http://127.0.0.1:9863/query
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 2.41µs, args: 
2022/10/08 17:43:17 GOOS duration: 94ns, args: 
2022/10/08 17:43:17 debug: Getenv

2022/10/08 17:43:17 Getenv duration: 1.831µs, args: WSL_DISTRO_NAME
2022/10/08 17:43:17 IsWsl duration: 4.519µs, args: 
2022/10/08 17:43:17 error: BatteryState
no battery
2022/10/08 17:43:17 BatteryState duration: 36.915µs, args: 
2022/10/08 17:43:17 debug: Pwd
/home/garcerant
2022/10/08 17:43:17 Pwd duration: 2.237µs, args: 
2022/10/08 17:43:17 GOOS duration: 105ns, args: 
2022/10/08 17:43:17 TemplateCache duration: 108ns, args: 
2022/10/08 17:43:17 TemplateCache duration: 170ns, args: 
2022/10/08 17:43:17 Shell duration: 179ns, args: 
2022/10/08 17:43:17 debug: Getenv

2022/10/08 17:43:17 Getenv duration: 3.874µs, args: XDG_CACHE_HOME
2022/10/08 17:43:17 CachePath duration: 36.38µs, args: 
2022/10/08 17:43:17 Flags duration: 97ns, args: 

Error: unknown command "shell" for "oh-my-posh"
Run 'oh-my-posh --help' for usage.

**Comment by JanDeDobbeleer** [comment]
@MiguelAngelGarcerantAmador I can't reproduce this. The latest version works flawlessly in bash, we also do not have the `oh-my-posh shell` command in our [bash init script](https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/src/shell/scripts/omp.bash). Are you certain there's nothing else in your `.bashrc`?
